src/components/v5/common/ActionSidebar/partials/motions/MotionSimplePayment/steps/ExitRecoveryStep/ExitRecoveryStep.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { FC } from 'react';
+import { formatText } from '~utils/intl';
+import Button from '~v5/shared/Button';
+
+import CardWithStatusText from '~v5/shared/CardWithStatusText';
+import ProgressBar from '~v5/shared/ProgressBar';
+import { ActionForm } from '~shared/Fields';
+import { ActionTypes } from '~redux';
+import MemberSignatureList from '~v5/common/MemberSignatureList';
+import { membersList } from './consts';
+import NotificationBanner from '~common/Extensions/NotificationBanner';
+import FormInputBase from '~v5/common/Fields/InputBase/FormInputBase';
+
+const displayName =
+  'v5.common.ActionSidebar.partials.motions.MotionSimplePayment.steps.ExitRecoveryStep';
+
+const ExitRecoveryStep: FC = () => {
+  return (
+    <div className="flex flex-col gap-4">
+      <CardWithStatusText
+        statusTextSectionProps={{
+          status: 'info',
+          children: formatText(
+            { id: 'motion.exitRecovery.statusText' },
+            { signatures: 2 },
+          ),
+          textClassName: 'text-4',
+          iconAlignment: 'top',
+          content: (
+            <ProgressBar
+              progress={0}
+              additionalText={formatText(
+                {
+                  id: 'motion.exitRecovery.additionalText',
+                },
+                { signatures: 2 },
+              )}
+            />
+          ),
+        }}
+        sections={[
+          {
+            key: '1',
+            content: (
+              <ActionForm
+                actionType={ActionTypes.ACTION_RECOVERY_EXIT}
+                className="flex flex-col gap-6"
+              >
+                <MemberSignatureList
+                  items={membersList}
+                  title={formatText({ id: 'common.memberSignature.title' })}
+                />
+                <Button
+                  mode="primarySolid"
+                  isFullSize
+                  type="submit"
+                  text={formatText({ id: 'motion.exitRecovery.submit' })}
+                />
+              </ActionForm>
+            ),
+          },
+        ]}
+      />
+      <CardWithStatusText
+        statusTextSectionProps={{
+          status: 'info',
+          children: formatText({
+            id: 'motion.exitRecovery.storageSlots.statusText',
+          }),
+          textClassName: 'text-4',
+          iconAlignment: 'top',
+        }}
+        sections={[
+          {
+            key: '1',
+            className: 'px-0 py-0',
```

Comment:
`p-0`

Reply:
I need to be more specific about the styles here, otherwise it overwrites the styles that are inside this component